app/src/main/java/com/example/coronadiagnosticapp/recorderExplanation.kt

```diff
@@ -0,0 +1,68 @@
+package com.example.coronadiagnosticapp
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import kotlinx.android.synthetic.main.fragment_recorder_explanation.*
+
+// TODO: Rename parameter arguments, choose names that match
+// the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+private const val ARG_PARAM1 = "param1"
+private const val ARG_PARAM2 = "param2"
+
+/**
+ * A simple [Fragment] subclass.
+ * Use the [recorderExplanation.newInstance] factory method to
+ * create an instance of this fragment.
+ */
+class recorderExplanation : Fragment() {
+    // TODO: Rename and change types of parameters
+    private var param1: String? = null
+    private var param2: String? = null
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        arguments?.let {
+            param1 = it.getString(ARG_PARAM1)
```

Comment:
this is the generic template that i have used, there is no "real" code here, i can removed it but since this was generated automatically i didn't messed with it.  Should i remove this?